sgd/sgd-aa-oas3.yml

```diff
@@ -0,0 +1,577 @@
+    get:
+      description: fdfd
+      operationId: get-delgation-service
+      summary:
+        "Returns the list of the delegations having the current user as
+        delegator and related to the services (or class of services) provided by
+        the authenticated RP and identifed from the service component in the URL"
+      security:
+        - dpopToken: []
+          dpopProof: []
+      x-spid-operation: # See §2.8
+        consentRequired: "SP"
+        spidLevel: SPID2
+        offlineAccess: false
+        offlineAccessDurationMax: 0
+      tags:
+        - protected
+      parameters:
+        - in: path
+          name: service
+          required: true
+          schema:
+            type: string
+          description:
+            Identifier of the service (or class of services) as given in the
+            online OAS3 catalog of delegable services provided by the
+            authenticated RP. It must be a UUID v4 identifier.
+          example: 21905575-991a-43d5-bede-25f59ae93373
+        - name: DPoP
+          in: header
+          required: true
+          description: DPoP proof
+          schema:
+            "$ref": "#/components/schemas/DPoP"
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DelegationList"
+              example:
+                delegation_list:
+                  - id: 1A00001AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-RSSMR*********I
+                  - id: 1A00002AA21
+                    https://attributes.eid.gov.it/fiscal_number: TINIT-ABCXX*********Z
+        "400":
+          $ref: "#/components/responses/400"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+        "429":
+          $ref: "#/components/responses/429"
+        "500":
+          $ref: "#/components/responses/500"
+
+  /delegation/get/{id}:
+    get:
+      operationId: get-delegation-id
+      summary: "
+        Returns the attributes related to the delegation identified by the
+        delegation_id component of the URL
+        "
+      security:
+        - dpopToken: []
+          dpopProof: []
+      x-spid-operation: # See §2.8
+        consentRequired: "SP"
+        spidLevel: SPID2
+        offlineAccess: false
+        offlineAccessDurationMax: 0
+      tags:
+        - protected
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+          description: Identitier of this delegation as defined by SGD
+        - name: DPoP
+          in: header
+          required: true
+          description: DPoP proof
+          schema:
+            "$ref": "#/components/schemas/DPoP"
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/jwt:
```

Comment:
Io non ho ancora chiaro come si associ uno schema al media type application/jwt. 

Jsonschema supporta qualcosa del genere https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.8.6 come estensione del tipo stringa. Probabilmente un tipo di supporto diretto in OAS può essere comodo. Non è banale però capire come un tool possa usare queste informazioni per generare gli stub.

Se vi metto in contatto con i tipi di OAS potete provare a seguire voi il filone? @fmarino-ipzs